plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts

```diff
@@ -150,6 +150,10 @@ store< MiniCart >(
 			},
 
 			openDrawer() {
+				if ( onCartClickBehaviour !== 'open_drawer' ) {
+					window.location.href = checkoutUrl;
+					return;
+				}
```

Comment:
I've seen that in the existing implementation they check `onCartClickBehaviour === 'navigate_to_checkout'`:  [link](https://github.com/woocommerce/woocommerce/blob/c41a27484cdc60ce2785aa632e1f5c8a7657e9bc/plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/block.tsx#L250-L253). I assume this works because "open_drawer" is the default value. Do you know if it could have other values and trigger the redirection when not wanted?

Reply:
I'm pretty sure its just the 2 but lets just change this to check the positive case :D

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/59098/commits/04c828496dd8598b87b2bba69aca4faebd8c343a